src/Acr.UserDialogs/Platforms/ios/AIDatePickerController.cs

```diff
@@ -306,6 +308,22 @@ private UIColor GetBackgroundColor()
                 return UIColor.White;
             }
         }
+
```

Comment:
@KoenDeleij you'll need to wrap this method in an #if __iOS__ directive, as `UIDatePickerStyle` isn't available in xamarin.tvos10. Let's get this merged and a new build! :) Thank you so much for your work :)

Reply:
Nor is it available on lesser version of iOS.  I haven't agreed to merge this, so don't overstep here please

Reply:
Right! I'll look into it

Reply:
@KoenDeleij tested the latest changes on iOS 13.5.1, and it looks good to me. Only nit I have is that the method should be `SetPreferredDatePickerStyle` (minor typo)

Reply:
Haha nice, I had to look twice to notice it.

Reply:
@KoenDeleji I've been doing some more testing of this using iOS 13.1 and 13.3, and it's still breaking with `NSInvalidArgumentException -[UIDatePicker setPreferredDatePickerStyle:]: unrecognized selector sent to instance 0x7fccd8e8e2a0`

I've tried a lot of different approaches including commenting out the call to `SetPreferredDatePickerStyle` to no avail. Any ideas what could be going wrong? I don't get it- there's no other place we're setting the style that I can see- commenting out your method should make it functionally identical to the currently published nuget package which DOES work...

Reply:
hmm thats odd. I've tested with a 13.1 and 14.0 implementation building the sourcecode directly into another project (not the sample project) without issues. 
Maybe it's caching? Is it breaking in the sample project? I'll try to reproduce it.

Reply:
The project does target iOS 14.

Reply:
I built it into a local nuget package and put that into another project (removing/deleting relevant obj/bin folders), and it's breaking (also targeting iOS 14).

I'll try building the sample project and see what happens

Reply:
Tested in the sample project and it's working fine there. I guess Visual Studio is holding onto the old package somewhere.

Reply:
Ah weird, must have been some caching.

Reply:
why this issue still is not merged? I see new release was 2 day ago...

Reply:
@pinkysek Go elsewhere with your attitude.  I have said I had not decided if I was going to take this yet.  

Have you ever contributed to this project or are you a sponsor?  Does this project owe you a thing?  Fork it and make your own build if you are so entitled.

Reply:
@aritchie I'm sorry man. I didn't want to be negative or something like that. I think at the moment is DatePicker on iOS 14.0 unusable so I thought that is critical issue. So I was confused with release 2 days ago without this fix. And I contributed already and I will definitely help more in the future.